### DISCUSSION TITLE
Is there any way to give custom equipment to guards by command?

### DISCUSSION THREAD

**Discussion by LyeZinho** [seymourimadeit/guardvillagers#236] [Q&A]
I have an small server so i want to make guards protect the spawn with some command blocks but i need to know if is there any way to change their equipment.

**Comment by seymourimadeit** [comment]
https://mcstacker.net/1.20.php

use this site, pick a vanilla mob id, copy and paste it in game, then replace the id with guardvillagers:guard before entering
